server-spi-private/src/main/java/org/keycloak/events/EventType.java

```diff
@@ -149,6 +149,8 @@ public enum EventType {
     DELETE_ACCOUNT(true),
     DELETE_ACCOUNT_ERROR(true),
 
+    USER_DISABLED_BY_PERMANENT_LOCKOUT(true),
```

Comment:
When calling `EventBuilder.error`, you should get an error because the `_ERROR` version of this enum is not available.

I know it does not make much sense to throw an error in this case but I think it is better to add the error enum for the sake of consistency.

Wdyt?

Reply:
@pedroigor I did not get your point. Can you provide more details? Thank you

Reply:
@pedroigor should be ok now.